.github/workflows/copilot_deploy.yml

```diff
@@ -208,62 +104,21 @@ jobs:
     uses: communitiesuk/funding-service-design-workflows/.github/workflows/post-deploy.yml@main
     with:
       run_performance_tests: ${{ inputs.run_performance_tests || true }}
-      run_e2e_tests: ${{ inputs.run_e2e_tests || false }}
-      environment: dev
+      run_e2e_tests_assessment: ${{ inputs.run_e2e_tests || false }}
+      run_e2e_tests_application: ${{ inputs.run_e2e_tests || false }}
```

Comment:
think it needs to have the new input names with appplication/assessment on the end